components/Dataservices/DescribeDataservice.vue

```diff
@@ -425,7 +425,6 @@
           <slot />
         </div>
       </div>
-      <div class="h-64" />
```

Comment:
This line is useful to have a little bit of margin bottom below the forms (it could be done with a margin-bottom on the container I think, it could be better but maybe keep it and change this in another PR?)

Reply:
Let's merge then :-) I will add it back if necessary